bindings/wasm/examples/public/examples.js

```diff
@@ -536,7 +539,37 @@ export const examples = {
         0, 0, size / Math.sqrt(2)
       ]);
       return result;
-    }
+    },
+
+    Split: function() {
```

Comment:
For better or worse, these serve double duty as JS tests, but primarily as public examples in ManifoldCAD.org. I'd like to keep that list highly curated. Would you mind adding the split functionality to one of the existing examples instead? I think `RoundedFrame` would be good - you can just split by a centered cube to make the inside a different color from the outside.

Reply:
OK, I split the frame into the inside and outside part. The inside part is slightly different color and appears/disappears as part of an animation. This makes it clear that it's a separate mesh and not just a material group on the surface of the frame.

The animation could be made a bit nice if moving from scale 1 to scale 0 wasn't animated. Is there a way to achieve this with your animation system? I could also do the following:
```ts
insideNode.translation = (t) => {
  const isInsideVisible = t < 0.5;
  return isInsideVisible ? [0,0,0] : [10000,10000,10000];
}
```
which looks better but it's a pretty nasty hack that probably should not be in the public examples

Reply:
No, I think what you have is good. Once animation pointer support is broadly adopted, we can probably do fancier things.